x-pack/plugins/remote_clusters/public/sections/remote_cluster_list/detail_panel/detail_panel.js

```diff
@@ -38,21 +39,126 @@ export class DetailPanelUi extends Component {
     cluster: PropTypes.object,
     closeDetailPanel: PropTypes.func.isRequired,
     clusterName: PropTypes.string,
+    copyCluster: PropTypes.func,
+  }
+
+  copyCluster = () => {
+    const { copyCluster, cluster } = this.props;
+    const {
+      name,
+      seeds,
+      skipUnavailable,
+    } = cluster;
+
+    const clusterConfig = {
+      name,
+      seeds,
+      skipUnavailable,
+    };
+
+    copyCluster(clusterConfig);
```

Comment:
I am surprised that eslint let us do this. At first sight, it seemed like a recursive call. 😊